munit/src/main/java/munit/Ignore.java

```diff
@@ -4,5 +4,5 @@
 import java.lang.annotation.Retention;
 
 @Retention(RetentionPolicy.RUNTIME)
-@interface Ignore {
+@interface IgnoreSuite {
```

Comment:
Dotty complains because there's already a `munit.Ignore` object (defined in Scala).

I actually think the error makes sense, so I took the liberty of renaming this

Reply:
Yeah, I can live with this. We should probably turn `val Ignore = new Tag("Ignore")` into `object Ignore extends Tag("Ignore")` so that it can be auto-imported in IDEs